lib/api/api.js

```diff
@@ -21,17 +22,26 @@ import objectPutPart from './objectPutPart';
 import serviceGet from './serviceGet';
 import vault from '../auth/vault';
 
+const RequestContext = policies.RequestContext;
 auth.setAuthHandler(vault);
 
 const api = {
-    callApiMethod(apiMethod, request, log, callback) {
+    callApiMethod(apiMethod, request, log, callback, locationConstraint) {
```

Comment:
as discussed, this would result in parameter reassignment and the locationConstraint argument after the callback is only used in the putBucket call.